### ISSUE TITLE
Add a Glossary or something similar

### ISSUE THREAD

**Issue by glozow** [bitcoin-core-review-club/website#621]
It's perhaps worth keeping a glossary so we don't have to continuously re-define things like ancestor feerate, block-relay-only connections, effective fees in coin selection, etc. We sometimes link to previous review clubs, but that also doesn't offer a direct definition. Maybe something similar to bitcoinops topics (so it's easily linkable and back-linkable) but for low-level terms.

**Comment by stickies-v** [comment]
Agreed on the concept. Intuitively I'd try to avoid adding another (fragmented) source of information, though. Most topics can be probably just be added to bitcoinops? But the very Core-specific items I can imagine don't belong there. Still, I think I'd prefer adding to existing knowledge bodies. [bitcoin-devwiki](https://github.com/bitcoin-core/bitcoin-devwiki/wiki) comes to mind, but... that's not very well organized atm.

**Comment by jnewbery** [comment]
> Most topics can be probably just be added to bitcoinops? But the very Core-specific items I can imagine don't belong there. Still, I think I'd prefer adding to existing knowledge bodies. [bitcoin-devwiki](https://github.com/bitcoin-core/bitcoin-devwiki/wiki) comes to mind, but... that's not very well organized atm.

For more high-level concepts that are user-visible or not specific to Bitcoin Core, I agree that we shouldn't be duplicating content here where we could just point to an external resource. However, the review club covers a lot of content that is either specific to Bitcoin Core internals (eg `ConnMan`), focused on protocol development (eg fuzz testing), or related to work-in-progress PRs/projects (eg package relay). I don't think there's an external resource that is appropriate for any of those things, and having a lightweight glossary on the site makes sense for those topics.
